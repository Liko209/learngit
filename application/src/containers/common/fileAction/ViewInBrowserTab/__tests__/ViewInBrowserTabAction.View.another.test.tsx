/*
 * @Author: Wayne Zhou (wayne.zhou@ringcentral.com)
 * @Date: 2019-08-28 09:38:44
 * Copyright Â© RingCentral. All rights reserved.
 */

import React from 'react';
import { ViewInBrowserTabAction } from '../ViewInBrowserTabAction.View';
import { shallow } from 'enzyme';
import { Translation } from 'react-i18next';

jest.mock('@/common/isUserAgent', () => ({ isElectron: true }));
describe('ViewInBrowserTabActionAnotherTest', () => {
  const wrapper = shallow(<ViewInBrowserTabAction />);
  it('should show correct text in electron [JPT-2926]', () => {
    expect(wrapper.find(Translation).shallow()).toMatchSnapshot();
  });
});
