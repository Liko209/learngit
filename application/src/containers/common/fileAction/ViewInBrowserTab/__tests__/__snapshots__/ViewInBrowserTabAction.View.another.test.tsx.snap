// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewInBrowserTabActionAnotherTest should show correct text in electron [JPT-2926] 1`] = `"message.fileAction.viewInBrowser"`;
